app/src/main/java/fr/iut_rodez/pathpilot_android_client/login/LoginService.java

```diff
@@ -0,0 +1,139 @@
+package fr.iut_rodez.pathpilot_android_client.login;
+
+import static fr.iut_rodez.pathpilot_android_client.util.Network.getRequestQueue;
+import static fr.iut_rodez.pathpilot_android_client.util.VolleyErrorHandler.handleError;
+
+import android.app.ProgressDialog;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Log;
+
+import com.android.volley.AuthFailureError;
+import com.android.volley.NetworkError;
+import com.android.volley.NoConnectionError;
+import com.android.volley.ParseError;
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.ServerError;
+import com.android.volley.TimeoutError;
+import com.android.volley.toolbox.JsonObjectRequest;
+import com.android.volley.toolbox.Volley;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Date;
+
+import fr.iut_rodez.pathpilot_android_client.BuildConfig;
+import fr.iut_rodez.pathpilot_android_client.util.Network;
+import fr.iut_rodez.pathpilot_android_client.util.Popup;
+
+public class LoginService {
```

Comment:
Missing Javadoc comment to describe what the activity is for

Reply:
FIx in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/38/commits/3f381088a74825447a65980cc8bb98e00dd60954